plugins/win-capture/get-graphics-offsets/d3d9-offsets.cpp

```diff
@@ -229,7 +229,7 @@ void get_d3d9_offsets(struct d3d9_offsets *offsets)
 					break;
 
 					/* check to make sure offsets actually point
-				 * toward expected data */
+					 * toward expected data */
```

Comment:
Again, this seems like the first line should be de-indented one level. Perhaps this should be placed inside the `ifdef` so that it is not separated from the section it's commenting about.

Reply:
De-indented one level and moved under the line `#ifdef _MSC_VER`.